inc/ocf_err.h

```diff
@@ -95,6 +95,25 @@ typedef enum {
 
 	/** Invalid cache line size */
 	OCF_ERR_INVALID_CACHE_LINE_SIZE,
+
+	/** Unable to perform operation due to concurrent management request */
+	OCF_ERR_MNGMT_OP_IN_PROGRESS,
```

Comment:
We need to have consistent naming for management related things. Sometime we use mngmt, sometimes mngt. We need to have one prefix.

Reply:
fixed this one, will add backlog item to track fixing of other instances.